src/instrumentation/health_check.go

```diff
@@ -0,0 +1,66 @@
+package instrumentation
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Health struct {
+	Healthy            bool               `json:"healthy"`
```

Comment:
these all need to be `yaml:` instead of `json:` since the sidecar is returning yaml
https://github.com/newrelic-experimental/k8s-apm-agent-health-sidecar/blob/main/internal/handler/health.go#L32

Reply:
fixed